Index: fnat-project/Scenes/Animatronic.tscn
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[gd_scene format=3 uid=\"uid://c603rgefo4g1i\"]\n\n[node name=\"Animatronic\" type=\"Node2D\"]\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fnat-project/Scenes/Animatronic.tscn b/fnat-project/Scenes/Animatronic.tscn
--- a/fnat-project/Scenes/Animatronic.tscn	(revision 5db500fa1b6e2f1999d9715dfab19841f10170d3)
+++ b/fnat-project/Scenes/Animatronic.tscn	(date 1762369146356)
@@ -1,3 +1,3 @@
-[gd_scene format=3 uid="uid://c603rgefo4g1i"]
+[gd_scene load_steps=0 format=3 uid="uid://c603rgefo4g1i"]
 
 [node name="Animatronic" type="Node2D"]
Index: fnat-project/Scenes/MainMenu.tscn
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[gd_scene format=3 uid=\"uid://db45n7f7blfpk\"]\n\n[node name=\"MainMenu\" type=\"Node2D\"]\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fnat-project/Scenes/MainMenu.tscn b/fnat-project/Scenes/MainMenu.tscn
--- a/fnat-project/Scenes/MainMenu.tscn	(revision 5db500fa1b6e2f1999d9715dfab19841f10170d3)
+++ b/fnat-project/Scenes/MainMenu.tscn	(date 1762369146358)
@@ -1,3 +1,3 @@
-[gd_scene format=3 uid="uid://db45n7f7blfpk"]
+[gd_scene load_steps=0 format=3 uid="uid://db45n7f7blfpk"]
 
 [node name="MainMenu" type="Node2D"]
Index: fnat-project/Scenes/SecurityDesk.tscn
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[gd_scene format=3 uid=\"uid://babct5l22vq0e\"]\n\n[node name=\"SecurityDesk\" type=\"Node2D\"]\n\n[node name=\"Camera2D\" type=\"Camera2D\" parent=\".\"]\n\n[node name=\"ColorRect\" type=\"ColorRect\" parent=\"Camera2D\"]\ncolor = Color(1, 0.007843138, 1, 1)\nmetadata/Pink = false\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fnat-project/Scenes/SecurityDesk.tscn b/fnat-project/Scenes/SecurityDesk.tscn
--- a/fnat-project/Scenes/SecurityDesk.tscn	(revision 5db500fa1b6e2f1999d9715dfab19841f10170d3)
+++ b/fnat-project/Scenes/SecurityDesk.tscn	(date 1762369146363)
@@ -1,4 +1,4 @@
-[gd_scene format=3 uid="uid://babct5l22vq0e"]
+[gd_scene load_steps=0 format=3 uid="uid://babct5l22vq0e"]
 
 [node name="SecurityDesk" type="Node2D"]
 
Index: fnat-project/Scenes/CameraFeed.tscn
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[gd_scene format=3 uid=\"uid://cc6yoh43ybaso\"]\n\n[node name=\"CameraFeed\" type=\"Node2D\"]\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fnat-project/Scenes/CameraFeed.tscn b/fnat-project/Scenes/CameraFeed.tscn
--- a/fnat-project/Scenes/CameraFeed.tscn	(revision 5db500fa1b6e2f1999d9715dfab19841f10170d3)
+++ b/fnat-project/Scenes/CameraFeed.tscn	(date 1762369146370)
@@ -1,3 +1,3 @@
-[gd_scene format=3 uid="uid://cc6yoh43ybaso"]
+[gd_scene load_steps=0 format=3 uid="uid://cc6yoh43ybaso"]
 
 [node name="CameraFeed" type="Node2D"]
Index: fnat-project/Scenes/GameOver.tscn
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[gd_scene format=3 uid=\"uid://w0vq0vaoqesw\"]\n\n[node name=\"GameOver\" type=\"Node2D\"]\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fnat-project/Scenes/GameOver.tscn b/fnat-project/Scenes/GameOver.tscn
--- a/fnat-project/Scenes/GameOver.tscn	(revision 5db500fa1b6e2f1999d9715dfab19841f10170d3)
+++ b/fnat-project/Scenes/GameOver.tscn	(date 1762369146373)
@@ -1,3 +1,3 @@
-[gd_scene format=3 uid="uid://w0vq0vaoqesw"]
+[gd_scene load_steps=0 format=3 uid="uid://w0vq0vaoqesw"]
 
 [node name="GameOver" type="Node2D"]
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1762369146378)
+++ b/.idea/.gitignore	(date 1762369146378)
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
Index: fnat-project/project.godot
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>; Engine configuration file.\n; It's best edited using the editor UI and not directly,\n; since the parameters that go here are not all obvious.\n;\n; Format:\n;   [section] ; section goes between []\n;   param=value ; assign values to parameters\n\nconfig_version=5\n\n[application]\n\nconfig/name=\"fnat-project\"\nrun/main_scene=\"uid://babct5l22vq0e\"\nconfig/features=PackedStringArray(\"4.5\", \"Forward Plus\")\nconfig/icon=\"res://icon.svg\"\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fnat-project/project.godot b/fnat-project/project.godot
--- a/fnat-project/project.godot	(revision 5db500fa1b6e2f1999d9715dfab19841f10170d3)
+++ b/fnat-project/project.godot	(date 1762369146384)
@@ -11,6 +11,5 @@
 [application]
 
 config/name="fnat-project"
-run/main_scene="uid://babct5l22vq0e"
 config/features=PackedStringArray("4.5", "Forward Plus")
 config/icon="res://icon.svg"
